crv/crvShape.cc

```diff
@@ -535,7 +521,10 @@ static int markEdgesToFix(Adapt* a, int flag)
 
   ma::Mesh* m = a->mesh;
   ma::Entity* e;
-  ma::Entity* edges[3];
+
+  // markEdges could have upto 6 edges marked!!!
+  /* ma::Entity* edges[3]; */
```

Comment:
for tets.  Are there any plans to expand this code to support hexs, pyramids or prisms?